app/src/main/java/rec/games/pokemon/teambuilder/MainActivity.java

```diff
@@ -29,6 +32,13 @@ protected void onCreate(Bundle savedInstanceState)
 		super.onCreate(savedInstanceState);
 		setContentView(R.layout.activity_main);
 
+		RequestOptions requestOptions = new RequestOptions()
+			.placeholder(R.drawable.ic_poke_unknown)
+			.placeholder(R.drawable.ic_poke_unknown)
+			.placeholder(R.drawable.ic_poke_unknown)
```

Comment:
Why three placeholders?

Reply:
Those are typos. I am going to fix that right now.

Reply:
I updated this with the actual code that I meant. 
https://github.com/OSU-CS492-W19/final-project-team-pikachu/blob/2d075baf9132b05b264f9d1c6e4df51108f6c637/app/src/main/java/rec/games/pokemon/teambuilder/MainActivity.java#L36-L39